src/DotLiquid.Tests/DropTests.cs

```diff
@@ -49,6 +52,20 @@ public override object BeforeMethod(string method)
             }
         }
 
+        internal class CancellationDrop : Drop
+        {
+            public ManualResetEventSlim Signal { get; } = new ManualResetEventSlim();
+
+            public string Halt
+            {
+                get
+                {
+                    Signal.Wait(Context.CancellationToken);

```

Comment:
I'm not familiar with Signal.Wait
Could you point me to the documentation or implementation of that?